azure-pipelines-stageprod.yml

```diff
@@ -0,0 +1,46 @@
+# Continuous integration (CI) triggers cause a pipeline to run whenever you push 
+# an update to the specified branches or you push specified tags.
+trigger:
+  tags:
+    include:
+    - release-*tags/*

```

Comment:
Any reason why this is different in the UI related repos i.e. why is this not `- release-*`?

Reply:
Oh, my error. It has to be - release-*

Reply:
Should be fixed now.